library/ui/src/main/java/com/google/android/exoplayer2/ui/SimpleExoPlayerView.java

```diff
@@ -293,6 +301,9 @@ public SimpleExoPlayerView(Context context, AttributeSet attrs, int defStyleAttr
 
     // Shutter view.
     shutterView = findViewById(R.id.exo_shutter);
+    if (shutterView != null) {
+      shutterView.setBackgroundColor(shutterColor);
```

Comment:
I think this defaults to setting the background color to transparent. I think it'll also clear any background drawable that's been set directly on the shutter as part of a custom layout for the view. No need to update this change. I'll merge as is and add a check so that `setBackgroundColor` is only called if the attribute is actually defined.